src/Frame.h

```diff
@@ -259,6 +259,23 @@ class Frame_REQUEST_FNF {
 };
 std::ostream& operator<<(std::ostream&, const Frame_REQUEST_FNF&);
 
+class Frame_METADATA_PUSH {
+ public:
+  static constexpr bool Trait_CarriesAllowance = false;
+
+  Frame_METADATA_PUSH() {}
+  Frame_METADATA_PUSH(FrameMetadata metadata)
+      : header_(FrameType::METADATA_PUSH, FrameFlags_METADATA, 0),
+      metadata_(std::move(metadata)) {}
```

Comment:
What should I put as default flag in ReactiveSocket::metadataPush? FrameFlags_EMPTY or FrameFlags_METADATA?